packages/deployment/app/cloudformation.template.yml

```diff
@@ -982,7 +982,7 @@ Resources:
     {{/if}}
       LaunchConfigurationName:
         Ref: CumulusContainerInstanceLaunch
-      MinSize: '0'
+      MinSize: 1
```

Comment:
it may make sense to make this configurable in the same way that `ecs.maxInstances` is configurable.